### PR TITLE
#187419185(Product-review): A buyer should be able to provide a review on a product they have bought successfully, rating+feedback

### PR DESCRIPTION
**Pull request by yvanddniyo**
**What does this PR do?**

This PR allow buyer who  successfully bought the product to provide review of it.

**Description of Task to be completed?**

User should be a buyer to review the products
User should be successfully  bought the products.

**How should this be manually tested?**

Clone the repository.
Checkout to the branch ft-product-review-#187419185
Run npm install to install dependencies.
Run npm run migrate and npm run seed
Run npm run dev to start the development server.
Open a web browser and navigate to the /docs endpoint (e.g., http://localhost:PORT/docs).
Verify that the Swagger UI loads correctly.
First authorize by logging in any user and use the token provided.
Explore the endpoints for reviews and verify that everything is working.

**What are the relevant pivotal tracker stories?**

[Deliveries #187419185]

### CODE REVIEW

**Review by MugemaneBertin2001 — APPROVED** [review]
LGTM

**Review by Heisjabo — APPROVED** [review]

**Review by Jovz19200 — APPROVED** [review]
Good work done there!!

**Comment by teerenzo** [comment]
PR title is wrong

File: src/sequelize/migrations/20240515125226-reviews.js [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+        primaryKey: true
+    },
+    userId: {
+        type: Sequelize.INTEGER,
+        allowNull: false
+    },
+    productId: {
+        type: Sequelize.INTEGER,
+        allowNull: false
+    },
```
Comment (teerenzo) [inline src/sequelize/migrations/20240515125226-reviews.js]: references?

File: src/sequelize/models/reviews.ts [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+        },
+        allowNull: false
+    },
+    productId: {
+        type: DataTypes.INTEGER,
+        references: {
+
+        },
+        allowNull: false
+    },
```
Comment (teerenzo) [inline src/sequelize/models/reviews.ts]: references?

**Review by teerenzo — CHANGES_REQUESTED** [review]

**Review by teerenzo — APPROVED** [review]

**Pull request merged**